packages/cluster/src/defaults.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    dhis2Version: '',
+    image: 'dhis2/core:{version}-latest-alpine',
+    port: 8080,
+    root: false,
```

Comment:
If this were `true` it wouldn't be a breaking change

Reply:
That's true. :thinking: